webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -340,7 +347,18 @@ const TableActionBar: React.FC<Props> = ({
         </Space>
       </Column>
       <Column align="right">
-        {!!toggleComparisonView && <Button onClick={toggleComparisonView}>Compare</Button>}
+        <Space>
+          {!!toggleComparisonView && <Button onClick={toggleComparisonView}>Compare</Button>}
+          <RadioGroup
+            iconOnly
+            options={[
+              { icon: 'grid', id: 'scroll', label: 'Scroll View' },
+              { icon: 'list', id: 'paged', label: 'Paged View' },
```

Comment:
nit: I'm not sure these icons read correctly at a glance. They have tooltips so it's not a huge issue, but still. Looking through our icons I don't see any that would be better choices though,

Reply:
Right.. I'm asking in design channel for a more proper icon but have not heard back yet